src/search/QuickSearchField.js

```diff
@@ -277,7 +284,11 @@ define(function (require, exports, module) {
     QuickSearchField.prototype._render = function (results, query) {
         this._displayedQuery = query;
         this._displayedResults = results;
-        this._highlightIndex = null;
+        if (this._shouldHighLightFirstIndex) {
+            this._highlightIndex = 0;
```

Comment:
Now that I think of it, wouldn't it be easier to go with `this._firstHighlightIndex`, where the value would default to null but `QuickOpen` would have `options._firstHighlightIndex` set to 0?